spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'liking photos' do
+  before do
+  	@user = User.create email: 'natso@gmail.com', password: '12345678', password_confirmation: '12345678'
```

Comment:
Line is too long. [104/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.